contrib/gitian/README.md

```diff
@@ -126,7 +126,7 @@ Setup for LXC:
 
 ```bash
 GH_USER=fluffypony
-VERSION=v0.16.0.0
+VERSION=v0.17.0.0
```

Comment:
The descriptor `.yml` files in /gitian should also get a bump from 0.16 to 0.17.